controls/SV-204444.rb

```diff
@@ -39,22 +38,22 @@
     Use the following command to map an existing user to the "user_u" SELinux user:
     $ sudo semanage login -m -s user_u <username>'
   impact 0.5
-  tag 'legacy': ['SV-86595', 'V-71971']
-  tag 'severity': 'medium'
-  tag 'gtitle': 'SRG-OS-000324-GPOS-00125'
-  tag 'gid': 'V-204444'
-  tag 'rid': 'SV-204444r792826_rule'
-  tag 'stig_id': 'RHEL-07-020020'
-  tag 'fix_id': 'F-4568r792825_fix'
-  tag 'cci': ['CCI-002165', 'CCI-002235']
+  tag legacy: ['SV-86595', 'V-71971']
+  tag severity: 'medium'
+  tag gtitle: 'SRG-OS-000324-GPOS-00125'
+  tag gid: 'V-204444'
+  tag rid: 'SV-204444r877392_rule'
+  tag stig_id: 'RHEL-07-020020'
+  tag fix_id: 'F-4568r792825_fix'
+  tag cci: ['CCI-002165', 'CCI-002235']
   tag nist: ['AC-3 (4)', 'AC-6 (10)']
-  tag subsystems: ["selinux"]
+  tag subsystems: ['selinux']
   tag 'host'
 
   if virtualization.system.eql?('docker')
     impact 0.0
-    describe "Control not applicable - SELinux settings must be handled on host" do
-      skip "Control not applicable - SELinux settings must be handled on host"
+    describe 'Control not applicable - SELinux settings must be handled on host' do
+      skip 'Control not applicable - SELinux settings must be handled on host'
     end
   else
 
```

Comment:
might just be cause it's late, but idk what especially that last one wants from me.  think the language for the other 3 is parsable but could be clearer

<img width="787" alt="image" src="https://github.com/mitre/redhat-enterprise-linux-7-stig-baseline/assets/2015821/708b9791-2377-4fd8-a278-fa14a6f575b5">

Reply:
I added an issue to explore this.